### ISSUE TITLE
Plans for RRF?

### ISSUE THREAD

**Issue by saswat0** [snexus/llm-search#82]
Hey @snexus 
Are there any plans of including an option for [RRF](Reciprocal Rank Fusion) along with Marco and BGE for reranking?

**Comment by snexus** [comment]
Hi @saswat0 

It crossed my mind as well, are you aware of any benefits of RRF compared to cross-encoder (besides speed)? In RRF, there are manual hyper-parameters that the user will need to adjust, such as weights dedicated to sparse vs dense results.

**Comment by saswat0** [comment]
@snexus I've found hardly any upside except computational efficiency. In staging setups, it gives us an additional degree of controllability (weighing) against various retrievers. But when using a non-embeddings based sparse retriever (BM25, tf-idf), I found RRF to be a better bet.

**Comment by snexus** [comment]
Makes sense, think it is worth implementing in sake of feature completeness, not as a priority though
